default.go

```diff
@@ -207,8 +207,11 @@ func (dt *Default) HTMLTemplate() string {
     /* Data table ------------------------------ */
     .data-wrapper {
       width: 100%;
-      margin: 0;
-      padding: 35px 0;
+      margin: 25px 0 25px 15px;
+      padding-top: 15px;
+    }
```

Comment:
Is this change in css is really necessary ? I don't really want to change the default theme unless it's necessary, as it could be considered as a regression for some people.

Reply:
I am guessing the idea here was to indent for the caption. I would inject a `margin-left` style in the template when caption Title field exists.